config/urls.py

```diff
@@ -31,4 +31,7 @@
     # Try to install the django debug toolbar, if exists
     if apps.is_installed("debug_toolbar"):
         import debug_toolbar
-        urlpatterns = [path("__debug__/", include(debug_toolbar.urls))] + urlpatterns
+
+        urlpatterns = [
+            path("__debug__/", include(debug_toolbar.urls))
+        ] + urlpatterns
```

Comment:
This is caused by running format. I assume black maybe?

Reply:
@tm-kn I don't think I ran the format command when I added this code, and since then I added format on save, so it got missed, and you've just picked it up.

I'll look at integrating format into CI.

Reply:
I never got round to adding it to the CI.

There are a set of config for formatting as part of the dev image that I intended to use everywhere: https://github.com/nationalarchives/docker/tree/main/docker/tna-python-dev/lib

The CI could either download and use the Docker image or maybe checkout the Docker repo so it can use those files.